code/espurna/config/hardware.h

```diff
@@ -2980,6 +2980,31 @@
     // No RGB on this bulb. Warm white on channel 0, cool white on channel 3
     #define MY92XX_MAPPING      255, 255, 255, 3, 0
 
+// -----------------------------------------------------------------------------
+// Lombex Lux Nova 2 White and Color
+// https://www.amazon.com/Lombex-Compatible-Equivalent-Dimmable-2700K-6500K/dp/B07B8K72PR
+// -----------------------------------------------------------------------------
+#elif defined(LOMBEX_LUX_NOVA2_WHITE_COLOR)
+
+    // Info
+    #define MANUFACTURER        "LOMBEX"
+    #define DEVICE              "LUX_NOVA2_WHITE_COLOR"
+    #define RELAY_PROVIDER      RELAY_PROVIDER_LIGHT
+    #define LIGHT_PROVIDER      LIGHT_PROVIDER_MY92XX
+    #define DUMMY_RELAY_COUNT   1
+
+    // Light
+    #define LIGHT_CHANNELS      5
+    #define MY92XX_MODEL        MY92XX_MODEL_MY9291
+    #define MY92XX_CHIPS        1
+    #define MY92XX_DI_PIN       4
+    #define MY92XX_DCKI_PIN     5
+    #define MY92XX_COMMAND      MY92XX_COMMAND_DEFAULT
+    // RGB on channels 0/1/2, warm white on channel 3
+    // The presence of warm leds on these bulbs may just be a common defect.
+    // If your bulb has cool leds, use 0, 1, 2, 3. Can also reduce LIGHT_CHANNELS.
+    #define MY92XX_MAPPING      0, 1, 2, 255, 3
```

Comment:
Hmmm. Upon reconsideration I think you are right, should just use LIGHT_CHANNELS 4 and MY92XX_MAPPING 0,1,2,3. I'll push up a fix.

LIGHT_CHANNELS 4 and MY92XX_MAPPING 0,1,2,3 works if people have bulbs with cool LEDS.
It might cause confusion in the interface though, since the web interface treats the 4th channel as cool and it might be warm. Probably would look weird if they enable "Use white channel" etc.

But as is with warm leds, features like "white color temperature" are kinda broken anyway. If we use LIGHT_CHANNELS 4 and MY92XX_MAPPING 0,1,2,3 at least people who do happen to have cool LEDS can have a properly working "Use white channel" feature .